crates/libs/core/src/runtime/mod.rs

```diff
@@ -4,35 +4,43 @@
 // ------------------------------------------------------------
 
 use mssf_com::{
-    FabricCommon::{
-        FabricRuntime::{
-            FabricCreateRuntime, FabricGetActivationContext, IFabricCodePackageActivationContext,
-            IFabricRuntime, IFabricStatefulServiceFactory, IFabricStatelessServiceFactory,
-        },
-        IFabricAsyncOperationCallback, IFabricAsyncOperationContext,
+    FabricCommon::FabricRuntime::{
+        FabricCreateRuntime, FabricGetActivationContext, IFabricCodePackageActivationContext,
+        IFabricRuntime,
     },
     FABRIC_ENDPOINT_RESOURCE_DESCRIPTION,
 };
 use windows_core::{Error, Interface, HSTRING, PCWSTR};
 
-use self::{
-    config::ConfigurationPackage, executor::Executor, stateful::StatefulServiceFactory,
-    stateful_bridge::StatefulServiceFactoryBridge, stateless::StatelessServiceFactory,
-    stateless_bridge::StatelessServiceFactoryBridge,
-};
+#[cfg(feature = "tokio_async")]
+use mssf_com::FabricCommon::{IFabricAsyncOperationCallback, IFabricAsyncOperationContext};
+
+use self::config::ConfigurationPackage;
 
+#[cfg(feature = "tokio_async")]
+pub use self::runtime::Runtime;
+
+#[cfg(feature = "tokio_async")]
 mod bridge;
 pub mod config;
 pub mod error;
+#[cfg(feature = "tokio_async")]
 pub mod executor;
+#[cfg(feature = "tokio_async")]
 pub mod node_context;
+#[cfg(feature = "tokio_async")]
```

Comment:
Could be simplified by using cfg-if crate: https://crates.io/crates/cfg-if

Reply:
Isn't that mostly for else-if type chainign?

Using a mod in this file might help to wrap them all up together

Reply:
You can use a mod and reexport it as well.